examples/convert_ieeg_to_bids.py

```diff
@@ -282,6 +292,7 @@
 raw = mne.io.read_raw_fif(op.join(
     misc_path, 'seeg', 'sample_seeg_ieeg.fif'))
 raw.info['line_freq'] = 60  # specify power line frequency as required by BIDS
+raw.set_channel_types({ch: 'ecog' for ch in raw.ch_names})  # fake intracranial
```

Comment:
why do you need this?

Reply:
It seems like it makes sense for it to be intracranial data in the raw file. This really should be checked by mne-bids that the data type matches too

Reply:
In the BIDSPath matches the raw file

Reply:
Why isn't the channel type set correctly automatically when reading the file with MNE? Is there a problem with the data? Or a bug with MNE?